internal/pkg/v2/infrastructure/redis/reading.go

```diff
@@ -97,6 +100,77 @@ func deleteReadingById(conn redis.Conn, id string) (edgeXerr errors.EdgeX) {
 	return nil
 }
 
+// AsyncDeleteReadings Handles the deletion of device readings asynchronously. This function is expected to be running
+// in a go-routine and works with the "deleteReadingsByIds" function for better performance.
+func (c *Client) AsyncDeleteReadings() {
```

Comment:
Move to Core Data app layer

Reply:
Per comments from Cloud above, I still leave this channel in the infra layer.